glide.lock

```diff
@@ -98,7 +98,7 @@ imports:
   version: 5bf94b69c6b68ee1b541973bb8e1144db23a194b
 - name: github.com/Sirupsen/logrus
   version: ba1b36c82c5e05c4f912a88eab0dcd91a171688f
-  repo: https://github.com/sirupsen/logrus
+  repo: https://github.com/Sirupsen/logrus
```

Comment:
As far as I can tell if you pull in a transitive dependency update where they have renamed their package import to use the lowercase version of `Sirupsen` then we must also use the lower case version because I dont think glide can handle that translation properly. That being said, changing repo to be upper case will probably break things in a more significant way.

We should probably decide immediately if we are going to port over to the lowercase version of sirupsen or keep trying to triage these issues